### ISSUE TITLE
train deeplabv3+ val result is to low

### ISSUE THREAD

**Issue by a5336033** [open-mmlab/mmsegmentation#3023]
When I train deeplabv3+ using the mmseg framework, the default setting of mmseg is to use 4 GPUs with a batch size of 4 on each GPU, training for 20,000 iterations. However, when you switch to training on a single GPU with a batch size of 4 and increase the iterations to 80,000, the resulting mIoU is only 63.35, which is nearly 12 points lower than the original paper's 75.93. The dataset used is VOC 2012 AUG. You would like to know the possible reasons for this？and learning rate needs to be adjusted？
